samza-core/src/main/java/org/apache/samza/clustermanager/container/placement/ContainerPlacementMetadataStore.java

```diff
@@ -259,6 +260,8 @@ static String toContainerPlacementMessageKey(UUID uuid, Class<?> messageType) {
         throw new SamzaException(e);
       }
     }
+    // Sort the actions in order of timestamp
+    newActions.sort(Comparator.comparingLong(ContainerPlacementRequestMessage::getTimestamp));
```

Comment:
Note to reviewers: Metastore does not give any metadata fields related to a message so I have to manually sort to ensure messages are in increasing order of timestamp